src/app/custom-pdp/custom-product-intro/custom-product-intro.component.html

```diff
@@ -5,7 +5,7 @@
         <span class="discount-price">$29.99</span>
         <span class="actual-price">$39.99</span>
         <span class="discount-percentage"> (25% off)   </span>
-        <span>Sold</span>
+        <span>Sold asdasdasdasdad</span>
```

Comment:
Delete this span "Sold asdasdasdasdad" and add a button to with "Add to Wishlist" text